ee/tabby-webserver/src/service/mod.rs

```diff
@@ -41,10 +41,20 @@ struct ServerContext {
     code: Arc<dyn CodeSearch>,
 }
 
+fn to_cli_args(url: String, token: String) -> Arc<[String]> {
+    vec!["--token".to_string(), token, "--url".to_string(), url].into()
```

Comment:
i think for now we could simply use `localhost:$port` as an intermediate solution. you could pass port in `attach_webserver`

Reply:
This wouldn't really simplify anything, since it already has the address and port being passed as a single parameter; all this would do is make it more inflexible. Why would we want that?